akka-stream-tests/src/test/scala/akka/stream/scaladsl/FlowMapAsyncUnorderedSpec.scala

```diff
@@ -121,8 +118,8 @@ class FlowMapAsyncUnorderedSpec extends AkkaSpec {
         .to(Sink(c)).run()
       val sub = c.expectSubscription()
       sub.request(10)
-      val expected = (OnComplete :: List(1, 2, 4, 5).map(OnNext.apply)).toSet
-      c.probe.receiveWhile(2.seconds, messages = 5) { case x ⇒ x }.toSet should be(expected)
+      val expected = (TestSubscriber.OnComplete :: List(1, 2, 4, 5).map(TestSubscriber.OnNext.apply)).toSet
```

Comment:
Would be nice to make this sugared, too.

    probe.expectEvents (ev1, ev2, ev3)

and/or the alternative

    probe.expectScript(action1 -> List(ev1, ev2), action2 -> Nil)